frontend/package.json

```diff
@@ -94,12 +94,12 @@
     "name-initials": "^0.1.3",
     "prop-types": "^15.6.2",
     "query-string": "^5.1.0",
-    "react": "^16.10.2",
+    "react": "^16.13.1",
```

Comment:
@f3ve  We really need this update right now? It can break the application, maybe we should address a react update in another issue